pkgs/applications/audio/google-play-music-desktop-player/default.nix

```diff
@@ -0,0 +1,86 @@
+{ stdenv, alsaLib, atk, cairo, cups, dbus, dpkg, expat, fontconfig, freetype
+, fetchurl, GConf, gdk_pixbuf, glib, gtk2, libpulseaudio, makeWrapper, nspr
+, nss, pango, udev, xorg
+}:
+
+let
+  version = "4.1.1";
+
+  deps = [
+    alsaLib
+    atk
+    cairo
+    cups
+    dbus
+    expat
+    fontconfig
+    freetype
+    GConf
+    gdk_pixbuf
+    glib
+    gtk2
+    libpulseaudio
+    nspr
+    nss
+    pango
+    stdenv.cc.cc
+    udev
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXrender
+    xorg.libXScrnSaver
+    xorg.libXtst
+  ];
+
+in
+
+stdenv.mkDerivation {
+  name = "google-play-music-desktop-player-${version}";
+
+  src = fetchurl {
+    # TODO: Replace 4.0.5 with ${version} on next version bump
+    url = "https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/releases/download/v${version}/google-play-music-desktop-player_4.0.5_amd64.deb";
+    sha256 = "11qqdhjbh60v9fvikkgnisqlad78670gq7rbm24160n06b0q7p4r";
+  };
+
+  buildInputs = [ dpkg makeWrapper ];
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  unpackPhase = ''
+    runHook preUnpack
+    dpkg -x $src .
+    runHook postUnpack
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    cp -r ./usr/share $out
+    cp -r ./usr/bin $out
+
+    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
+             "$out/share/google-play-music-desktop-player/Google Play Music Desktop Player"
```

Comment:
Hmm.. I can't seem to get this working. [Here](http://lpaste.net/4557365007234367488) is the updated derivation after replacing `phases` with `dontBuild` and cutting out the manual patching.

First, it doesn't seem to be changing the interpreter, so nothing happens when I run it:
~~~bash
$ patchelf --print-interpreter $(which google-play-music-desktop-player)
/lib64/ld-linux-x86-64.so.2
~~~

If I add that manual step back in, then it can't find the libraries:
~~~bash
$ google-play-music-desktop-player
google-play-music-desktop-player: error while loading shared libraries: libgtk-x11-2.0.so.0: cannot open shared object file: No such file or directory
~~~

Am I doing something wrong?

In case you didn't notice, this is a pre-built binary. The Spotify package was doing something similar, but then they may have other reasons for manual patching.